test/typing/reveal/namedtuple.py

```diff
@@ -9,7 +9,8 @@
 t_sort.values[0, 0] == 1.5  # noqa: B015
 reveal_type(t_sort)  # E: Tuple[{Tensor}, {Tensor}, fallback=torch.return_types.sort]
 
-t_qr = torch.linalg.qr(t)
-t_qr[0].shape == [2, 2]     # noqa: B015
-t_qr.Q.shape == [2, 2]      # noqa: B015
-reveal_type(t_qr)  # E: Tuple[{Tensor}, {Tensor}, fallback=torch.return_types.qr]
+# TODO: activate when linalg has typing
+# t_qr = torch.linalg.qr(t)
+# t_qr[0].shape == [2, 2]     # noqa: B015
+# t_qr.Q.shape == [2, 2]      # noqa: B015
+# reveal_type(t_qr)  # E: Tuple[{Tensor}, {Tensor}, fallback=torch.return_types.qr]
```

Comment:
Based on the analysis here https://github.com/pytorch/pytorch/pull/103376#issuecomment-1588203198